pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -44,23 +44,25 @@ func filterAntreaConns(conns []*flowexporter.Connection, nodeConfig *config.Node
 		srcIP := conn.TupleOrig.SourceAddress
 		dstIP := conn.TupleReply.SourceAddress
 
-		// Only get Pod-to-Pod flows.
+		// Consider Pod-to-Pod, Pod-To-Service and Pod-To-External flows.
 		if srcIP.Equal(nodeConfig.GatewayConfig.IP) || dstIP.Equal(nodeConfig.GatewayConfig.IP) {
-			klog.V(4).Infof("Detected flow through gateway :%+v", conn)
+			klog.V(4).Infof("Detected flow through gateway %s :%+v", nodeConfig.GatewayConfig.IP.String(), conn)
```

Comment:
A lot of flows go through the gateway. I think a more appropriate phrasing is probably "Detected flow for which one of the endpoint is the host gateway"?